src/express-route/azext_express_route/_params.py

```diff
@@ -24,6 +24,7 @@ def load_arguments(self, _):
     er_circuit_name_type = CLIArgumentType(options_list=('--circuit-name',), metavar='NAME', help='ExpressRoute circuit name.', id_part='name', completer=get_resource_name_completion_list('Microsoft.Network/expressRouteCircuits'))
     er_gateway_name_type = CLIArgumentType(options_list=('--gateway-name',), metavar='NAME', help='ExpressRoute gateway name.', id_part='name', completer=get_resource_name_completion_list('Microsoft.Network/expressRouteGateways'))
     er_port_name_type = CLIArgumentType(options_list=('--port-name',), metavar='NAME', help='ExpressRoute port name.', id_part='name', completer=get_resource_name_completion_list('Microsoft.Network/expressRoutePorts'))
+    er_bandwidth_type = CLIArgumentType(options_list='--bandwidth', nargs='+', help='Bandwidth of the circuit. Should be Mbps for service providers or Gbps for ExpressRoute ports. Usage: INT {Mbps,Gbps}')
```

Comment:
You don't need the "Should be..." statement. If you accept the unit you convert it to whatever the service requires. You will want to call out the defaults though (which maybe was what you were doing but it doesn't read that way).

Reply:
I left the original wording in case some thought was put behind it. Will remove that part.